test/extended/images/info.go

```diff
@@ -33,10 +33,10 @@ var _ = g.Describe("[sig-imageregistry][Feature:ImageInfo] Image info", func() {
 			set -x
 
 			# display info about an image on quay.io
-			oc image info quay.io/coreos/etcd:latest
+			oc image info quay.io/openshift-release-dev/ocp-release:4.18.3-x86_64
```

Comment:
fyi, there is also `exutilimage.ShellImage()` that is a more general image for such things and is able to be pulled from the cluster internal registry. 

not sure there is something worth testing specifically coming from quay.io though...